allensdk/brain_observatory/behavior/nwb/extensions/event_detection/extension_builder.py

```diff
@@ -0,0 +1,53 @@
+import os.path
+
+from pynwb.spec import NWBNamespaceBuilder, export_spec, NWBGroupSpec, NWBDatasetSpec
+
+NAMESPACE = 'ndx-event-detection'
+
+
+def main():
+    # these arguments were auto-generated from your cookiecutter inputs
+    ns_builder = NWBNamespaceBuilder(
+        doc="""Store the event detection output""",
+        name=f"""{NAMESPACE}""",
+        version="""0.1.0""",
+        author=list(map(str.strip, """Adam Amster""".split(','))),
+        contact=list(map(str.strip, """aamster@alleninstitute.org""".split(',')))
```

Comment:
You can change this to Wayne's e-mail, unless you want to potentially field questions about the NWB file from end users.

See:
https://github.com/AllenInstitute/AllenSDK/blob/64dd27a0525ecc8b6237cf5b488646790e48359e/allensdk/brain_observatory/nwb/ndx-aibs-behavior-ophys.namespace.yaml#L3